src/components/MovieCard/MovieCard.jsx

```diff
@@ -1,5 +1,50 @@
 import './MovieCard.scss';
 
-export const MovieCard = () => (
-  <>Put the card here</>
+export const MovieCard = ({ movie }) => (
+  <>
+    <div className="card" data-cy="Movie">
```

Comment:
```suggestion
  <div className="card" data-cy="Movie">
```
There is no need to use React fragment, as you already have a container for all children (card)